src/test/v1.2/advanced/fdc3.open.ts

```diff
@@ -20,26 +21,52 @@ export default () =>
     getCommonOpenTests(control, openDocs, config);
 
     //run 1.2-only tests
+
+    const AOpensB1 = `(AOpensB1) Can open app B from app A with no context and ensure that the correct app was opened`;
+    it(AOpensB1, async () => {
+      let targetApp: TargetApp;
```

Comment:
you don't really need this - since target app _could be_ a string and the next line makes it specifically a string